src/foam/u2/borders/BackgroundCard.js

```diff
@@ -14,19 +14,30 @@ foam.CLASS({
       width: 100%;
       padding: 2.4rem;
       border-radius: 4px;
+      overflow: auto;
+    }
+    ^.disablePadding {
+      padding: 0;
     }
   `,
 
   properties: [
     {
       class: 'Color',
       name: 'backgroundColor'
+    },
+    {
+      class: 'Boolean',
+      name: 'padding',
```

Comment:
Why not make this a string prop and just use that to style the border that way we can pass in something like `2.4rem` or `1.4rem 2rem` or `10px 12px 23px 34px`

Reply:
In my mind the intended purpose of this padding property is to disable legacy behavior of BackgroundCard, not to configure it.

Reply:
Dont see why background card needs to have a padding property at all, should be another border or a combined border like we discussed on slack

Reply:
yes that's what I'm saying